app/reducers/index.js

```diff
@@ -0,0 +1 @@
+export {default} from 'ember-redux/reducers/index';
```

Comment:
yes - it would throw an error letting you know the reducers/index.js was missing (not great but it also was a solid entry point into "starting using redux ..."). This improves it as we don't break out of the box (when this PR lands)